### ISSUE TITLE
vqvae can't reconstruction

### ISSUE THREAD

**Issue by LonglongaaaGo** [rosinality/vq-vae-2-pytorch#30]
Hi! thanks for your beautiful repo!
Recnetly, i have confused by the vqvae reconstruction. The code like below. I just input img and expect to get the same output, but it not. What is the matter with it?

            _, _, _, _id_t, _id_b = vqvae.encode(img)
            decoded_sample = vqvae.decode_code(_id_t, _id_b)
            decoded_sample = decoded_sample.clamp(-1, 1)
            save_image(decoded_sample, '%s/epoch_%dreal.png' % (opt.outf, epoch), normalize=True, range=(-1, 1))
         
thank you for reply !

**Comment by LonglongaaaGo** [comment]
There are some details :
1. I just use the pre-training model from this repo
2.the image of output is nothing in it, only dark pixel

**Comment by LonglongaaaGo** [comment]
sorry it is my mistak to forget include the pre-trained  model file 。。。

**Comment by SURABHI-GUPTA** [comment]
@LonglongaaaGo how did you get top and bottom latent representations ?
